### PR TITLE
provide default urls in site_actions (default profile)

### PR DESCRIPTION
**Pull request by nileshgulia1**

### CODE REVIEW

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

**Comment by tisto** [comment]
@nileshgulia1 can you elaborate on why we need to set this in plone.volto?

**Comment by nileshgulia1** [comment]
@tisto We need generic setup amendment as per https://github.com/plone/volto/pull/3248#issuecomment-1113140944 so that we have default footer links( contact-us, accessibility ) in siteActions URL .

File: src/plone/volto/profiles/default/actions.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<object xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        meta_type="Plone Actions Tool"
+        name="portal_actions"
+>
+  <action-provider name="portal_workflow" />
+  <action-provider name="portal_types" />
+  <action-provider name="portal_actions" />
```
Comment (avoinea) [inline src/plone/volto/profiles/default/actions.xml:8]: @nileshgulia1 Not needed.

File: src/plone/volto/profiles/default/actions.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      <property name="url_expr">string:${globals_view/navigationRootUrl}/contact-form</property>
+      <property name="icon_expr">string:plone-contact-info</property>
+      <property name="available_expr" />
+      <property name="permissions">
+        <element value="View" />
+      </property>
+      <property name="visible">True</property>
+      <property name="modal"
+                type="text"
+      >{}</property>
```
Comment (avoinea) [inline src/plone/volto/profiles/default/actions.xml:70]: I'm not sure this is needed, either :thinking:

**Review by avoinea — CHANGES_REQUESTED** [review]

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

File: src/plone/volto/profiles/default/actions.xml (hunk 1 above)
Comment (nileshgulia1) [reply src/plone/volto/profiles/default/actions.xml]: done [2ff8128](https://github.com/plone/plone.volto/pull/64/commits/2ff81286ac357d80e7b4b0898ef2e9d284fb76f7)

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

**Review by avoinea — APPROVED** [review]

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by davisagli — COMMENTED** [review]
I would like to see this do a more targeted update of the action's `url_expr` only. In other words, `title`, `description`, `icon_expr`, `available_expr`, `permissions`, and `visible` should not be set here, because it could overwrite customizations that have been done through the web.

**Comment by sneridagh** [comment]
@nileshgulia1 could you take care of @davisagli suggestion? Thanks!

**Comment by sneridagh** [comment]
@nileshgulia1 ping! Could you please take a look at the suggestion above?

So the idea is that the profile only touches the `url_expr`, and remove the rest.

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
@davisagli please take a final look.

**Review by davisagli — CHANGES_REQUESTED** [review]
@nileshgulia1 @avoinea I'm looking at this more closely now, and I'm afraid I don't understand what you're trying to do.

This is changing, for example, the sitemap action URL from
`<property name="url_expr">python:f"{plone_portal_state.navigation_root_url()}/sitemap"</property>` (from https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/profiles/default/actions.xml#L50)
to
`<property name="url_expr">string:${globals_view/navigationRootUrl}/sitemap</property>`

It produces exactly the same URL as before, but now it is using a deprecated way to get the navigation root URL.

The sitemap and contact info actions are already returning valid Volto URLs, even without this change. Only the accessibility-info one needs to be fixed -- either by adding a Volto route for /accessibility-info, or by removing the action here.

**Comment by davisagli** [comment]
@nileshgulia1 @avoinea Oh, I see that at least the URL for the contact form is changing. I opened #93 as an updated version of this PR, so I'll close this one, but please let me know if I am still missing something.

**Pull request closed without merging**